packages/common/src/content/_internal/ContentSchema.ts

```diff
@@ -23,6 +23,15 @@ export const ContentSchema: IConfigurationSchema = {
     },
     schedule: {
       type: "object",
+      properties: {
+        _forceUpdate: {
+          type: "array",
+          items: {
+            type: "boolean",
+            enum: [true],
```

Comment:
Do we want to restrict this to only allow true? And how come we're using an array instead of a const?

Reply:
Yeah, I agree this feels a little strange, but it has to do with how the tile-select field works - when it's in "checkbox" mode (e.g. multiselect), we emit an array of values. 

Not to change things, but I did just have another idea which would be to use the switch field instead. This is configured to simply emit a `true`/`false` boolean rather than an array: 

<img width="500" alt="Screenshot 2024-05-02 at 1 44 57 PM" src="https://github.com/Esri/hub.js/assets/17460610/deb2b9c7-aba7-4c67-9519-b0be156c44de">
<img width="200" alt="Screenshot 2024-05-02 at 1 46 20 PM" src="https://github.com/Esri/hub.js/assets/17460610/72fd7168-4f1a-4310-a044-0f3b476967d6">

Not sure how design/Tomas would feel about this, but from the dev perspective it's a bit easier/more consistent to deal with